lib/typescript/httpclient/endpoints/getTags.ts

```diff
@@ -0,0 +1,43 @@
+import {doFetchFromBackend} from '../api';
+import {Tag} from '../model';
+import {GetTagsResponse} from '../payload';
+
+const colorMapper = (color: string): string => {
+  switch (color) {
+    case 'BLUE':
+      color = 'tag-blue';
+      break;
+    case 'RED':
+      color = 'tag-red';
+      break;
+    case 'GREEN':
+      color = 'tag-green';
+      break;
+    case 'PURPLE':
+      color = 'tag-purple';
+      break;
+    default:
+      color = 'tag-blue';
+  }
+  return color;
+};
+
+const tagsMapper = (serverTags: Tag[]): Tag[] => {
+  const tags: Tag[] = [];
+  const _ = serverTags.map((tag: Tag) => {
+    tag.color = colorMapper(tag.color);
+    tags.push(tag);
+  });
+  return tags;
+};
+
+export function getTags() {
```

Comment:
i'd rename in style of the conversations changes. `listTags` (same for the payloads)